test/sanity/params.yaml

```diff
@@ -0,0 +1,2 @@
+server: 127.0.0.1
```

Comment:
change to this:
```
source: "//127.0.0.1/share"
```

the original `server`, `share` parameter is only for NFS driver.

Reply:
Tried with both `source: "//127.0.0.1/share"` and `source: "//127.0.0.1/sambashare"` but still failing.

Reply:
Using `source: "//127.0.0.1/samba"` works for `prow-tests` but fails for `github workflow` tests.

`source: "//127.0.0.1/share"` was failing both the tests.